src/libs/pulumi-cli.ts

```diff
@@ -12,6 +12,25 @@ export async function isAvailable(): Promise<boolean> {
   return res.stderr != '' && !res.success ? false : res.success;
 }
 
+export async function getVersion(): Promise<string | undefined> {
+  core.debug('Executing command \'pulumi version\' to check version of CLI on the runner');
+  
+  const res = await exec.exec('pulumi', ['version'], true);
+
+  core.debug(`Success: ${res.success}`)
+  core.debug(`StdOut: ${res.stdout}`)
+  core.debug(`StdErr: ${res.stderr}`)
```

Comment:
Let's either drop these log lines, or provide a little more context about where they're running so they can be more easily differentiated from similar lines during debugging. If they were useful to you while debugging, I imagine they'd be useful to us too.

Reply:
Even though they did help me debug the `ExecResult` and figure out what to use in the `if` statement, I'm fine with removing them. I started with your suggestion of checking `stderr === ''` but discovered that didn't work using the debug logs. I made sure to add a comment block explaining that so I think the debug logs have served their purpose and are not needed anymore.